flac.go

```diff
@@ -30,6 +30,8 @@ import (
 
 	"github.com/mewkiz/flac/frame"
 	"github.com/mewkiz/flac/meta"
+
+	"github.com/mikkyang/id3-go/encodedbytes"
```

Comment:
Should we start using `vgo` to track versions of external dependencies? We can do this in a follow-up PR.

Reply:
Sounds good, I'll create the issue.